### PR TITLE
Add hello world to tutorial

### LINKED ISSUE #20: Needs quick 'hello world' to get app running.
I usually like to have apps running while i read the documentation. But in the [tutorial](http://hack.aragon.org/docs/tutorial.html), the process for getting the app running starts with

> Let's first set up our project.
>
> `npm i -g @aragon/cli`
>
> Next, we bootstrap our project:
>
> `aragon init foo.aragonpm.eth`

and then goes into many pages of documentation. Towards the bottom of the page, we see `aragon run`. It also doesn't mention `npm install` at all, which I forgot to do at first.

### PR DESCRIPTION
**Pull request by jtremback**
Close #20

### CODE REVIEW

File: docs/tutorial.md [hunk 1]
```diff
@@ -6,6 +6,18 @@ title: Tutorial
 # Your first Aragon app
 In this guide, we will walk you through creating your first Aragon app using [aragonOS](os-ref.md), [aragon.js](js-ref.md), [Aragon UI](ui-intro.md) and [Aragon CLI](https://github.com/aragon/aragon-cli).
 
+# Quickstart
+
+```
+npm i -g @aragon/cli
+aragon init foo.aragonpm.eth react
+cd foo
+npm install
```
Comment (izqui) [inline docs/tutorial.md:15]: As explained https://github.com/aragon/aragon-cli/issues/148#issuecomment-399354226, `npm install` shouldn't be required as it is already performed by the init command.

File: docs/tutorial.md [hunk 2]
```diff
@@ -6,6 +6,18 @@ title: Tutorial
...
+npm start
```
Comment (izqui) [inline docs/tutorial.md:16]: I would substitute `npm start` with `aragon run` to be consistent with the rest of the documentation.

**Review by izqui — COMMENTED** [review]
Thanks for your contribution, this will definitely help! Just a couple of details, and as soon as the CLA is accepted by you, we can merge.

**Comment by luisivan** [comment]
Applied all changes by @izqui

**Pull request merged**